src/megatron/energon/av/fastseek/keyframeinfo.py

```
@@ -17,15 +17,3 @@ class KeyframeInfo:
 
     index: int
```

Suggest to make it `int | None`, so we don't need to use `-1` to mark unavailable. This will also make code fail early that relies on integers here but get's a matroska video.